lib/kitchen/provisioner/chef_base.rb

```diff
@@ -47,18 +48,6 @@ class ChefBase < Base
         providers/**/* recipes/**/* resources/**/* templates/**/*
       ].join(",")
 
-      default_config :chef_metadata_url do |provisioner|
```

Comment:
yeah, although it's generated from `chef_omnibus_url`, so one can set that and both windows and unixes will do the right thing.